data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg

```diff
@@ -1126,7 +1131,7 @@
 
                 [message]
                     speaker=Hidel
-                    message= _ "We have heard much of your intelligence and courage. The number of humans over the centuries who have earned the respect and admiration of the Northern Elves are extremely few, but you are one of them."
+                    message= _ "We have heard much of your intelligence and courage. The number of humans over the centuries who have earned the respect and admiration of the northern elves are extremely few, but you are one of them."
```

Comment:
Original was correct, it's a proper noun. Possibly the wrong one, but that's a discussion in #6624.